### PR TITLE
upgrade to redux 5 and RTK 2 + minor packages updates

### PR DESCRIPTION
**Pull request by gshilin**

### CODE REVIEW

**Review by davgur — APPROVED** [review]

File: .yarn/install-state.gz
Comment (bbfsdev) [inline .yarn/install-state.gz:1]: Do we want .yarn/install-state.gz to be part of git?
Is this the correct way to go? (I don't know)

File: src/components/Sections/Library/LibraryContainer.js [hunk 1]
```diff
@@ -375,7 +373,7 @@ class LibraryContainer extends Component {
   };
 
   switchSortingOrder = parentId => {
-    const { sortBy, NotToSort } = this.props;
+    const { sortBy,  } = this.props;
```
Comment (bbfsdev) [inline src/components/Sections/Library/LibraryContainer.js:376]: Remove "," and one space.

File: yarn.lock
Comment (bbfsdev) [inline yarn.lock:1]: Should this be part of github?

**Review by bbfsdev — APPROVED** [review]

File: .yarn/install-state.gz
Comment (gshilin) [reply .yarn/install-state.gz:1]: me too

File: yarn.lock
Comment (gshilin) [reply yarn.lock:1]: yes, of course

File: src/components/Sections/Library/LibraryContainer.js (hunk 1 above)
Comment (gshilin) [reply src/components/Sections/Library/LibraryContainer.js]: done

**Pull request merged**